src/v2/view-builder/views/phone/ChallengeAuthenticatorPhoneOptionsView.js

```diff
@@ -0,0 +1,118 @@
+import {
+  loc,
+  createButton } from 'okta';
+import BaseView from '../../internals/BaseView';
+import BaseForm from '../../internals/BaseForm';
+import BaseFooter from '../../internals/BaseFooter';
+import BaseFactorView from '../shared/BaseFactorView';
+import { addSwitchAuthenticatorLink } from '../../utils/AuthenticatorUtil';
+
+const Body = BaseForm.extend(Object.assign(
+  {
+    className: 'phone-authenticator-challenge',
+    subtitle: ' ',
+
+    title () {
+      return loc('oie.phone.verify.title', 'login');
+    },
+
+    noSubmitButton: true,
+
+    saveForm () {
+      BaseForm.prototype.saveForm.call(this, this.model);
+    },
+
+    initialize () {
+      BaseForm.prototype.initialize.apply(this, arguments);
+      const primaryMode = this.model.get('primaryMode');
+      const secondaryMode = this.model.get('secondaryMode');
+      const primaryButtonTitle = (primaryMode === 'sms')
+        ? loc('oie.phone.sms.primaryButton', 'login')
+        : loc('oie.phone.call.primaryButton', 'login');
+
+      this.add(
+        createButton({
+          attributes: {
+            'data-se': 'phone-authenticator-challenge__button--primary',
+            'type': 'button'
+          },
+          className: 'button button-primary phone-authenticator-challenge__button--primary',
+          title: primaryButtonTitle,
+          click: function () {
+            // Call the API to send a code via primary mode
+            this.model.set('authenticator.methodType', primaryMode);
+            this.saveForm.call(this, arguments);
+          }.bind(this),
+        })
+      );
+      if (secondaryMode) {
+        const secondaryButtonTitle = (secondaryMode === 'sms')
+          ? loc('oie.phone.sms.secondaryButton', 'login')
+          : loc('oie.phone.call.secondaryButton', 'login');
+        this.add(
+          createButton({
```

Comment:
Given both are just buttons but different styles, did you consider to do followings (utilize save and button bar)
```
save() {
  return isPrimayMode === 'sms' ? xxxx : yyyy;
}

render() {
  super.render();

  if sencodaryMode {
    const secondaryBtn = ...;
     this.add(secondaryBtn, '.o-form-button-bar');
  }
```

Reply:
^ did not work, I tried the following - 
* `postRender` hook in the `form` itself - same issue
*  Overriding `postRender` hook in the `BaseFactorView`, but that puts it after footer links.
* I can probably include it in footer links with a className for styling, but that seems hacky. It's cleaner to have it in the form IMHO.

Please let me know if there's anything else I can try